src/extractor/guidance/segregated_intersection_classification.cpp

```diff
@@ -203,15 +219,17 @@ std::unordered_set<EdgeID> findSegregatedNodes(const NodeBasedGraphFactory &fact
         return info;
     };
 
-    auto const isSegregatedFn = [&](auto const &edgeData,
+    auto const isSegregatedFn = [&](EdgeID edgeID,
```

Comment:
We use `under_score` convention when naming variables. So that would be `edge_id` and `edge_data`.

Reply:
Done